app/src/main/java/org/oppia/android/app/databinding/ConstraintViewAdapters.java

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.app.databinding;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.constraintlayout.widget.ConstraintSet;
+import androidx.databinding.BindingAdapter;
+
+/** Holds all custom binding adapters that set constraints to views. */
+public class ConstraintViewAdapters {
+
+  /** Binding adapter for setting a [layout_constraintEndToEndOf] to a view */
+  @BindingAdapter("app:layout_constraintEnd_toEndOf")
+  public static void setConstraintEndToEndOf(
+      @NonNull View view, int constraintToId
```

Comment:
I believe we can type this view to be a ``ConstraintLayout``.

Reply:
Are we saying that we don't need ```View view``` ?

Reply:
@BenHenning Here we are setting some constraints to the children of the `ConstraintLayout`, not to the `ConstraintLayout` itself, and the children of the `ConstraintLayout` can be any `View` type.

Reply:
Ah, good point. Please disregard this comment then.